roles/buildbot-worker/tasks/main.yml

```diff
@@ -9,6 +9,7 @@
       - python-pip
       - ostree
       - virtualenv
+      - subversion
```

Comment:
I think ideally this would be before `virtualenv` to maintain the sorting, but meh. I'll leave this in approved and you can either push a fixup or merge as is.